src/quo/components/list_items/approval_info/view.cljs

```diff
@@ -0,0 +1,118 @@
+      [:tag-label {:optional true} [:maybe :string]]
+      [:on-button-press {:optional true} [:maybe fn?]]
+      [:on-avatar-press {:optional true} [:maybe fn?]]
+      [:on-option-press {:optional true} [:maybe fn?]]
+      [:container-style {:optional true} [:maybe :any]]]]]
+   :any])
+
+(defn- avatar
+  [{:keys [type avatar-props blur? theme on-press]}]
+  [rn/pressable {:on-press on-press}
+   (cond
+     (#{:spending-cap :token-contract :network
+        :collectible}
+      type)                         [token-avatar/view
+                                     (assoc avatar-props
+                                            :type
+                                            (if (= type :collectible) :collectible :asset))]
+     (= type :account)              [account-avatar/view (assoc avatar-props :size 32)]
+     (= type :collectible-contract) [collection-avatar/view (assoc avatar-props :size :size-32)]
+     (= type :spending-contract)    [dapp-avatar/view avatar-props]
+     (= type :date-signed)          [icon-avatar/icon-avatar
+                                     (assoc avatar-props
+                                            :size  :size-32
+                                            :color (if blur?
+                                                     colors/white
+                                                     (colors/theme-colors colors/neutral-50
+                                                                          colors/neutral-40
+                                                                          theme)))]
+     (= type :address)              [wallet-user-avatar/wallet-user-avatar
+                                     (assoc avatar-props
+                                            :size       :size-32
+                                            :monospace? true
+                                            :lowercase? true
+                                            :neutral?   true)]
+     (= type :community)            [community-avatar/view avatar-props])])
+
+(defn- view-internal
+  [{:keys [type avatar-props label description blur? unlimited-icon? container-style
+           on-option-press on-avatar-press on-button-press button-label button-icon tag-label
+           option-icon]}]
+  (let [theme        (quo.theme/use-theme)
+        description? (not (string/blank? description))]
+    [rn/view
+     {:style               (merge (style/container description? blur? theme) container-style)
+      :accessibility-label :approval-info}
+     [avatar
+      {:type         type
+       :blur?        blur?
+       :theme        theme
+       :on-press     on-avatar-press
+       :avatar-props avatar-props}]
+     [rn/view
+      {:style style/labels}
+      [rn/view
+       {:style {:flex-direction :row :align-items :center}}
+       [text/text
+        {:size   :paragraph-1
+         :weight :semi-bold
+         :style  (style/label blur? theme)} label]
```

Comment:
Quick one: There's at least another instance of this style problem in this namespace https://guide.clojure.style/#vertically-align-fn-args

Reply:
I think this is something our linter should handle, no?

Reply:
@BalogunofAfrica, we would all love to get that automated, but it's a bit hard I think. I could try myself to configure this, but it's not trivial because we could easily introduce formatting side-effects in other parts of the code because we told zprint to respect newlines. Also for zprint, this is nothing more than a vector, so I think the solution could be to somehow tell it to detect if the second argument to a vector is a map and then try to fit it on the same line as the first element.

We could also try to consult with zprint's author.

Reply:
Fixed all of the formatting issues that I found.